README.md

```diff
@@ -235,6 +253,93 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Optional Arguments
+
+* -c (Config file)
+
+Use the config file specified.
+
+The default locations that will be checked are ```/etc/pump.io.json``` and
+```~/.pump.io.json```, use this option to override it. The default config
+values are stored in lib/defaults.js.
```

Comment:
The location of the defaults doesn't seem like something that needs to be in the README, especially since the defaults are documented in this file already.